src/world/area_kkj/kkj_11/ACE9E0.c

```diff
@@ -6,7 +6,12 @@
 
 #include "world/common/GetPeachDisguise.inc.c"
 
-INCLUDE_ASM(s32, "world/area_kkj/kkj_11/ACE9E0", func_80240F44_ACF084);
+ApiStatus func_80240F44_ACF084(void) {
+    GameStatus *temp_v1 = gGameStatusPtr;
```

Comment:
Left-align pointers (applies throughout PR)
```suggestion
    GameStatus* temp_v1 = gGameStatusPtr;
```